rewrite-gradle/build.gradle.kts

```diff
@@ -49,6 +49,9 @@ dependencies {
 tasks.withType<ShadowJar> {
     configurations = listOf(project.configurations.compileClasspath.get())
     archiveClassifier.set(null as String?)
+    minimize {
+        exclude("org/slf4j/impl/*.class")
+    }
```

Comment:
This doesn't do what you think it does. 

When tell the shadowjar plugin to `minimize`, it attempts to determine which classes are actually used by our code and  omit unused files from the final jar.
But we load this stuff at runtime, so _everything_ from the gradle dependencies is determined to be unused. Except, for some reason, package-info.java files which get to stay. 

By "excluding" slf4j you exclude it from being minimized, so it would actually be _retained_ through the minimization if the filter you supplied worked "correctly".  So you accidentally succeed at excluding slf4j because minimize ends up excluding _everything_. 

This isn't problematic for the change plugin recipe you ran since our [RewriteGradleProject](https://github.com/openrewrite/rewrite/blob/81b3bf010a4525c3b404cfbf86146abf13134312/rewrite-gradle/src/main/resources/RewriteGradleProject.groovy#L161) stub fully specifies the `plugins` block. So even with nothing from the gradle API we can parse that successfully. A recipe that operated on the `dependencies` or `repositories` blocks would not be so fortunate. 

Whenever you fiddle with bundling always open up and inspect the resulting bundle to make sure your fiddling achieves what you _think_ it achieves. 

Also, there are two separate sets of slf4j classes that sneak into rewrite-gradle. One from [rewrite-core](https://ge.openrewrite.org/s/ujjcj6o4zqgje/dependencies?dependencies=slf4j&expandAll) and one from gradle. So this should be a lot closer to what you want:
```suggestion
    exclude("org/slf4j/impl/**.class")
    exclude("org/gradle/internal/logging/slf4j/**.class")
```